vault/dendron.release.2021-11-09.md

```diff
@@ -0,0 +1,92 @@
+---
+id: EjPEcAkZIDgzHaVMk4teA
+title: Release Notes (version 0.68)
+desc: ''
+updated: 1636484692932
+created: 1636478484110
+---
+
+Dendron 0.68 has sprouted  🌱
+
+Take a look at the latest [Dendron wiki](https://wiki.dendron.so), and you'll see some new features directly from these highlights:
+
+* Local TOC in the right-hand nav for each page in the published docs
+* Newly combined search bar that can go directly to a note, or search note contents using the `?` prefix
+
+`Dendron: Show Preview` used to add some extra latency when rendering the preview and also made features like copy/paste unstable. This change migrates the old iframe-based preview to be a native webview. It brings the following changes:
+
+* Faster webview performance
+* Clipboard support (copy and paste works)!
+* Live color theme updates (vs reloading your window for changes to take effect)
+* Bundle native preview with webpack
+* Build native preview on `yarn setup`
+* Cleanup of assets 
+
+### Highlights
+* feat(schemas): adding new command - create schema from hierarchy ([[docs|dendron.topic.schema.create-from-hierarchy]])
+* feat(views): native dendron preview
```

Comment:
i know i wrote the entry but i realize this line by itself doesn't mean much.  can you rephrase this entry to
```
feat(views): better webviews - faster loading and more stable performance
```